Tasks/DeleteFilesV1/Tests/L0.ts

```diff
@@ -8,7 +8,7 @@ const testRoot = path.join(__dirname, 'test_structure');
 const removeFolder = function(curPath) {
     if (fs.existsSync(curPath)) {
         fs.readdirSync(curPath).forEach((file, index) => {
-        const newPath = path.join(path, file);
+        const newPath = path.join(curPath, file);
```

Comment:
Is this intentional change? Is it some fix for test here?

Reply:
Yes, it seems to be a typo that did not cause build errors before or other errors during test execution (it strange to my mind). But after bumping of the dependencies versions it started throwing an error during build. And in fact _'path'_ is an imported module and path.join(_path_, file) looks like a meaningless construction and should produce error.